bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -1137,5 +1165,6 @@ func (s *ksServer) StartHttp(port int) {
 	// add an http handler for prometheus metrics
 	http.Handle("/metrics", promhttp.Handler())
 
+	go countHeartbeat()
```

Comment:
nit: you could probably inline this one as

```go
go func() {
   time.Sleep(10 * time.Second)
   serviceHeartbeat.Inc()
}()
```

Reply:
Sorry, added the labels too soon...

Reply:
It works the same either way :)